client/my-sites/site-settings/disconnect-site/index.jsx

```diff
@@ -16,27 +16,41 @@ import { getSelectedSite } from 'state/ui/selectors';
 import Main from 'components/main';
 import Placeholder from 'my-sites/site-settings/placeholder';
 import redirectNonJetpack from 'my-sites/site-settings/redirect-non-jetpack';
+import NavigationBackButton from 'my-sites/site-settings/navigation-back-button';
 import SkipSurvey from './skip-survey';
 
 class DisconnectSite extends Component {
+	getRoute() {
+		const { siteSlug } = this.props;
+
+		if ( siteSlug ) {
+			return '/settings/manage-connection/' + siteSlug;
+		}
+	}
+
 	render() {
 		const { site, translate } = this.props;
 
 		if ( ! site ) {
 			return <Placeholder />;
 		}
 		return (
-			<Main className="disconnect-site site-settings">
-				<DocumentHead title={ translate( 'Site Settings' ) } />
-				<FormattedHeader
-					headerText={ translate( 'Disconnect Site' ) }
-					subHeaderText={ translate(
-						'Tell us why you want to disconnect your site from WordPress.com.'
-					) }
-				/>
-				<Card className="disconnect-site__card"> </Card>
-				<SkipSurvey />
-			</Main>
+			<div>
+				<span className="disconnect-site__back-button">
+					<NavigationBackButton redirectRoute={ this.getRoute() } { ...this.props } />
```

Comment:
Seems passing `...this.props` is not necessary.

Reply:
@seear, this is to pass e.g. `translate` as a prop, since the component is not connected. Is there a better way to go about?

Reply:
Ah, ok! The call to [`localize()`](https://github.com/Automattic/wp-calypso/pull/17691/files#diff-2faed744e1c939d8d389c63dc7bc8f5bR33) wraps a component with the [`localize`](https://github.com/Automattic/i18n-calypso/tree/dceace781efd139fbf62c3fd6719113bac4e2f87/lib/localize) higher-order-component, which has some useful props, one of which is [`translate`](https://github.com/Automattic/i18n-calypso/blob/dceace781efd139fbf62c3fd6719113bac4e2f87/lib/localize/index.js#L13), meaning we don't need to pass it down.